setup.py

```diff
@@ -38,6 +48,9 @@
         'Topic :: Internet :: WWW/HTTP',
         'Topic :: Text Processing :: Markup :: HTML',
     ],
-    install_requires=['numpy', 'w3lib', 'six', 'cython'],
-    ext_modules=cythonize(extensions),
+    install_requires=['numpy', 'w3lib', 'six'],
+    extras_require={
+        'speedup': ['cython']
```

Comment:
What is it for? If an user installs `scrapely[speedup]` there won't be any speedup for the user, right?